src/open_prime_hunters_rando/file_manager.py

```diff
@@ -1,16 +1,30 @@
+from enum import Enum
 from pathlib import Path
 
+import construct
 from construct import Container, ListContainer
 from ndspy.rom import NintendoDSRom
 
 from open_prime_hunters_rando.entities.entity_type import EntityFile
 from open_prime_hunters_rando.level_data import get_data
+from open_prime_hunters_rando.string_tables.string_tables import StringTable
+
+
+class Language(Enum):
+    ENGLISH = "stringTables"
+    FRENCH = "stringTables_fr"
+    GERMAN = "stringTables_gr"
+    ITALIAN = "stringTables_it"
+    JAPANESE = "stringTables_jp"
+    SPANISH = "stringTables_sp"
 
 
 class FileManager:
     def __init__(self, rom: NintendoDSRom):
         self.rom = rom
         self.entity_files: dict[str, EntityFile] = {}
+        self.string_tables: dict[str, StringTable] = {}
+        construct.lib.setGlobalPrintFullStrings(True)
```

Comment:
AFAIK this is a debug function and shouldnt be in production.

Reply:
If there's a better way to be able to see the fully exported file without truncation then I'm all ears. Not sure how else to avoid this.

Reply:
Waiting on #82 to be merged first